Library/ViewModels/RewardCardContainerViewModel.swift

```diff
@@ -171,7 +171,7 @@ private func pledgeButtonIsEnabled(project: Project, reward: Reward) -> Bool {
   let isAvailable = rewardIsAvailable(reward: reward)
   let isBacking = userIsBacking(reward: reward, inProject: project)
 
-  return (project.state == .live && isAvailable) || isBacking
+  return (project.state == .live && isAvailable && !isBacking) || (project.state != .live && isBacking)
 }
```

Comment:
"Note that other Reward states (such as View Pledge, Fix pledge, etc) are not changing here, because they will be handled from the new ManagePledgeViewController." **Noted!**